### DISCUSSION TITLE
Update README?

### DISCUSSION THREAD

**Discussion by SpaceWhite** [github/securitylab#54] [General] [answered]
Hi guys

Securitylab readme provides very little information about the detail of this repository
Don't you guys think adding more details on the readme would be great?

**Comment by xcorail** [comment, marked as answer]
Hey @SpaceWhite 
We'll soon transform this README into something more useful.
I cannot spoil the surprise, you'll have to 👀 it.
cc @kevinbackhouse

**Comment by kevinbackhouse** [comment]
It's looking a bit better [now](https://github.com/github/securitylab/tree/59bd6967a4421a8591cd1b8bb30a34950fc7766c).
